spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -140,8 +136,129 @@ definitions:
             desc: Height
 
  - MSG_EPHEMERIS:
+    id: 0x0047
+    short_desc: WGS84 satellite ephemeris parameters
```

Comment:
s/WGS84 satellite ephemeris parameters/Satellite broadcast ephemeris/
and delete reference to WGS84 from long description